app/build.gradle

```diff
@@ -1,18 +1,17 @@
 apply plugin: 'com.android.application'
-
 apply plugin: 'kotlin-android'
-
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 26
+    compileSdkVersion versions.androidSdk.compile
+    buildToolsVersion versions.androidSdk.buildTools
+
     defaultConfig {
-        applicationId "com.niamhpower.fay"
-        minSdkVersion 23
-        targetSdkVersion 26
+        applicationId 'com.niamhpower.fay'
```

Comment:
groovy typical is single-quotes unless it needs to be double quotes (needs to be when it's gonna replace some token within the string, see dependencies.gradle)